apps/betterangels-backend/clients/tests/test_queries.py

```diff
@@ -112,16 +112,23 @@ def test_client_profiles_query(self) -> None:
 
         query = f"""
             query ViewClientProfiles {{
-                clientProfiles{{
-                    {self.client_profile_fields}
+                clientProfilesPaginated(pagination: {{offset: $offset, limit: $limit}}) {{
```

Comment:
I think this test can be left as is, since `test_client_profiles_paginated_query`, below, is testing the paginated query